tests/unit/test_utils.py

```diff
@@ -826,3 +827,15 @@ def test_import_member_no_such_class():
             tmt.utils.GeneralError,
             match=r"No such member 'NopeDoesNotExist' in module 'tmt\.steps\.discover'."):
         tmt.plugins.import_member('tmt.steps.discover', 'NopeDoesNotExist')
+
+
+# IMPORTANT: This test should run as last, as it reimports the `tmt.utils`
+# module and breaks imports imported from `tmt.utils` namespace to this module
+def test_workdir_env_var(tmpdir, monkeypatch):
+    """ Test TMT_RUN_DIR environment variable """
+    with monkeypatch.context() as ctx:
+        ctx.setenv('TMT_RUN_DIR', str(tmpdir))
+        importlib.reload(tmt.utils)
```

Comment:
Do not reimport. Fix implementation instead.

Reply:
agreed